eclipse-scout-core/src/menu/Menu.ts

```diff
@@ -731,4 +732,15 @@ export class Menu extends Action implements MenuModel {
     }
     return false;
   }
+
+  protected override _tooltipText(): string {
```

Comment:
Can you move the declaration of textVisibleOrig  from Action to Menu? It seems to be at the wrong place.